src/main/java/uk/gov/register/providers/ClientErrorExceptionMapper.java

```diff
@@ -35,13 +32,18 @@ public Response toResponse(ClientErrorException exception) {
 
     private ThymeleafView getEntityFor(ClientErrorException exception) {
         if (exception instanceof BadRequestException) {
-            return viewFactory.badRequestExceptionView((BadRequestException) exception);
+            return viewFactory.exceptionBadRequestView(exception.getMessage());
         }
 
         if (exception instanceof NotFoundException || exception instanceof NotAcceptableException) {
-            return viewFactory.thymeleafView("404.html");
+            return viewFactory.exceptionNotFoundView();
         }
 
-        throw exception;
+        if (exception instanceof NotAllowedException) {
+            return viewFactory.exceptionView("Method not allowed", "This method is not allowed");
```

Comment:
Either if it appears more than once or it is very common like 400 which can lead to it being used multiple times in the future :)